.github/secrel/config.yml

```diff
@@ -10,21 +10,38 @@
 #   - -Dexample-build-arg=some-value
 #   - -Dexample.maven=${{ secrets.example_secret }}
 
+snyk:
+  additional-args:
+  - -PGITHUB_ACCESS_TOKEN=${{ secrets.ACCESS_TOKEN }}
+
 # Remove image block if you're not building Docker images
 images:
-- name: app-image
-  path: "./Dockerfile"
-  args:
-  - example-build-arg=some-value
-  - example-build-arg-secret=${{ secrets.example_secret }}
-# Additional images can be specified like below
-# - name: 
-#   path: ""
-#   args:
-#   -
+  - name: abd_vro
+    path: "./Dockerfile"
+    args:
+      -  GITHUB_ACCESS_TOKEN=${{ secrets.ACCESS_TOKEN }}
+  - name: abd_vro-db-init
+    context: "./db-init/src/main/resources"
```

Comment:
This context value allows `./gradlew docker` to work.

Reply:
SecRel is works as well: https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/2974902421